app/views/contexts/section_summary_context.py

```diff
@@ -40,11 +40,28 @@ def __call__(
     def get_page_title(
         self, current_location: Location, title_for_location: str
     ) -> str:
+
+        section_repeating_page_title = (
+            self._schema.get_repeating_page_title_for_section(
+                current_location.section_id
+            )
+        )
+
         if custom_page_title := self._schema.get_custom_page_title_for_section(
             current_location.section_id
         ):
+            custom_page_title = (
+                f"{custom_page_title}: {section_repeating_page_title}"
+                if section_repeating_page_title
+                else custom_page_title
+            )
             return self._resolve_custom_page_title(custom_page_title, current_location)
 
+        title_for_location = (
+            f"{title_for_location}: {section_repeating_page_title}"
+            if section_repeating_page_title
+            else title_for_location
+        )
```

Comment:
It doesn't matter if there is a custom page title or not, the `section_repeating_page_title` should have a `{list_item_position}` placeholder that needs resolving.

Reply:
Would that be the case if there was not a repeat in a section?